src/Services/NonceService.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Services;
+
+const FIVE_MINUTES = 300;
+
+class NonceService {
+    private SessionService $sessionService;
+
+    public function __construct(SessionService $sessionService)
+    {
+        $this->sessionService = $sessionService;
+    }
+
+    public function generateNonce(string $uuid): string {
+        $session = $this->sessionService->getSession($uuid);
+        $nonce = bin2hex(random_bytes(8));
+        $session->set('nonce', [$nonce, time()]);
+        $session->set('hello', 'abc123');
```

Comment:
What is this for?

Reply:
Oh that's a mistake. It must've gotten left from when I was debugging the issues